src/Client.js

```diff
@@ -0,0 +1,57 @@
+// @flow
```

Comment:
Not convinced this file is really required. It really just allows you to pass a single variable around instead of all three.
I'm not fussed though